force-app/main/default/flexipages/Deal_Support_Process_Page_Single_Related_Lists.flexipage-meta.xml

```diff
@@ -117,29 +117,6 @@
                 <identifier>RecordStatus_cField</identifier>
             </fieldInstance>
         </itemInstances>
-        <itemInstances>
-            <fieldInstance>
-                <fieldInstanceProperties>
-                    <name>uiBehavior</name>
-                    <value>none</value>
-                </fieldInstanceProperties>
-                <fieldItem>Record.Closed_Reason__c</fieldItem>
-                <identifier>RecordClosed_Reason_cField</identifier>
-                <visibilityRule>
-                    <booleanFilter>1 OR 2</booleanFilter>
-                    <criteria>
-                        <leftValue>{!Record.Status__c}</leftValue>
-                        <operator>EQUAL</operator>
-                        <rightValue>Done - Successful</rightValue>
-                    </criteria>
-                    <criteria>
-                        <leftValue>{!Record.Status__c}</leftValue>
-                        <operator>EQUAL</operator>
-                        <rightValue>Closed - Lost</rightValue>
-                    </criteria>
-                </visibilityRule>
-            </fieldInstance>
-        </itemInstances>
         <itemInstances>
```

Comment:
@wchuprosci -- Did you just add this today? I'm guessing we don't want to remove it

Reply:
Yup these were part of my changes to add the naming logic for training events, please keep this in the layout

Reply:
This change is now restored, per commit 08bdd03. Thank you both!